src/vivarium_cluster_tools/psimulate/worker/vivarium_work_horse.py

```diff
@@ -80,20 +90,23 @@ def work_horse(job_parameters: dict) -> pd.DataFrame:
         logger.info(f'Average step length was {exec_time["step_mean_seconds"]:.3f} seconds.')
 
         sim.finalize()
-        metrics = sim.report(print_results=False)
+        sim.report(print_results=False)
         event["end"] = time()
         end_snapshot = CounterSnapshot()
-
         do_sim_epilogue(start_snapshot, end_snapshot, event, exec_time, job_parameters)
 
+        results = sim.get_results()  # Dict[measure, results dataframe]
+
         idx = pd.MultiIndex.from_tuples(
             [(job_parameters.input_draw, job_parameters.random_seed)],
             names=["input_draw_number", "random_seed"],
         )
-        output_metrics = pd.DataFrame(metrics, index=idx)
-        for k, v in collapse_nested_dict(job_parameters.branch_configuration):
-            output_metrics[k] = v
-        return output_metrics
+        finished_results_metadata = pd.DataFrame(index=idx)
```

Comment:
Why are you putting the "input_draw_number" and "random_seed" in the index here, when in `processing._concat_metadata` you drop the index?

Reply:
Will look into this